.travis.yml

```diff
@@ -56,57 +59,31 @@ before_install:
   - TOP=$(pwd)
   - "HCNUMVER=$(${HC} --numeric-version|perl -ne '/^(\\d+)\\.(\\d+)\\.(\\d+)(\\.(\\d+))?$/; print(10000 * $1 + 100 * $2 + ($3 == 0 ? $5 != 1 : $3))')"
   - echo $HCNUMVER
-  - CABAL="$CABAL -vnormal+nowrap+markoutput"
+  - CABAL="$CABAL -vnormal+nowrap"
   - set -o pipefail
-  - |
```

Comment:
Are these (and all other changes not directly related to GHC 8.10) warranted?

Reply:
Well I had to use a more recent version of `haskell-ci` that supports 8.10. These changes are due to the differences between that new version and the old version originally used to generate the file. There are probably some beneficial changes in there too…